rust-examples/operators.rs

```diff
@@ -120,6 +120,9 @@ mod wasm {
     }
 }
 
+// This function will be available to other Modules.
+// And can also be called by the interpreter at Module start (binaryen-shell -e test) 
+#[wasm_export]
```

Comment:
I don't see any why it wouldn't, no. It would probably only impact one's ability to control the wasm output. I don't know if I'd want all my pub fns to be exported in the wasm (esp for size reasons) if I'm not going to use all of them from another module (possibly from another rust module compiled separately, so a bit harder to optimize by binaryen).
In any case, I was doing imports and start, and it made testing easier with the interpreter, nothing is written in stone :)